doc/scenario/provide_custom_icons_for_the_project_or_item_type.md

```diff
@@ -29,11 +29,11 @@ In order to add custom icons to your project, you need to follow these steps:
   tree = tree.SetIcon(Images1Monikers.ProjectIconImageMoniker.ToProjectSystemType());
   ```
 
-##Adding an image manifest manually
+## Adding an image manifest manually
 1. Image manifest
   1. Create a new xml file, name it to .imagemanifest (e.g. `Images1.imagemanifest`)
   
-    ```xml
+```xml
```

Comment:
This indentation was intentional, as it nests the code block under the bullet point.

Reply:
Ok , but the problem is that the code block is not displayed correctly when there is an indent.

Reply:
I see what went wrong. The indentation within a list must be a further three spaces for the next level. I added a commit that fixes the indentation throughout the document.